reference/supervisor.md

```diff
@@ -81,6 +81,7 @@ scored.
 
 WbNodeRef wb_supervisor_node_get_from_def(const char *def);
 WbNodeRef wb_supervisor_node_get_from_id(int id);
+const char *wb_supervisor_node_get_def(WbNodeRef node);
```

Comment:
I feel it's better here, the first two are about getting nodes based on def/id, the following two (get_def and get_id) are about getting the def/ids themselves. I feel like the current layout makes more sense.